state-sync/storage-service/server/src/lib.rs

```diff
@@ -47,11 +56,17 @@ impl<T: StorageReaderInterface> StorageServiceServer<T> {
         request: StorageServiceRequest,
     ) -> Result<StorageServiceResponse, Error> {
         let response = match request {
+            StorageServiceRequest::GetAccountStatesChunkWithProof(request) => {
+                self.get_account_states_chunk_with_proof(request)
+            }
             StorageServiceRequest::GetEpochEndingLedgerInfos(request) => {
                 self.get_epoch_ending_ledger_infos(request)
             }
```

Comment:
how do we plan to get the latest ledger info inside the last epoch? do we need a `get_state_proof` kind of API as well?

Reply:
Aah, I was originally thinking about including this in the state sync summary, but I think it might make more sense as a dedicated API! I'll follow up with a PR here 😄